common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -1315,8 +1315,12 @@ def test_expand_all_when_some_collapsed(self):
             And all sections are expanded
         """
         self.course_outline_page.visit()
+        # We have seen unexplainable sporadic failures in this test. Try disabling animations to see
+        # if that helps.
+        self.browser.execute_script("jQuery.fx.off = true;")
```

Comment:
We have a helper method for this.
from common.test.acceptance.tests.helpers import disable_animations